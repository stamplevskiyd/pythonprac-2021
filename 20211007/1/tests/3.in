(0,1), (0,1), (5,6), (10,11), (3,4), (9,15), (122,834), (122,835), (200,300)
