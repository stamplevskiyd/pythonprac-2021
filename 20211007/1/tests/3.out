((122, 835), (200, 300))
